feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/domain/input/NumberInputField.kt

```diff
@@ -3,7 +3,7 @@ package app.k9mail.feature.account.setup.domain.input
 import app.k9mail.core.common.domain.usecase.validation.ValidationError
 import app.k9mail.core.common.domain.usecase.validation.ValidationResult
 
-data class NumberInputField(
+class NumberInputField(
```

Comment:
Why? When making such changes, please include the rationale in the commit message.

Reply:
The data class adds the copy method, which allows any changes of the input field. This could result in unwanted changes, as happened in #6951.